reporthandling/results/v1/reportsummary/datastructures.go

```diff
@@ -31,6 +32,18 @@ type FrameworkSummary struct {
 	ResourceCounters ResourceCounters    `json:",inline"`
 }
 
+// FrameworkSummary summary of scanning from a single framework perspective
+type FrameworkSubsectionSummary struct {
+	Name          string                       `json:"name"`         // subsection name
+	Framework     string                       `json:"framework"`    // framework name
+	ID            string                       `json:"id"`           // subsection id
+	ControlsStats map[apis.ScanningStatus]uint `json:"controlStats"` // control statistics
```

Comment:
This might cause issues->
What if you want the number of failed controls of two subFWs? You cannot get the right number because the same control might repeat itself. This is why in the FWs we  added the controls IDs using the relevant interfaces.

Reply:
You right. But this is a summary of the results of a subsection. If we want to do some manipulation on the data, we always can go back to the data itself and make this manipulation